extensions/wikia/AdEngine3/src/slots.js

```diff
@@ -27,6 +19,15 @@ function isIncontentBoxadApplicable() {
 		!context.get('wiki.targeting.wikiIsCorporate');
 }
 
+/**
+ * Enables top_boxad on screen with width >= 1024px.
+ *
+ * @returns {boolean}
+ */
+function isTopBoxadEnabled() {
```

Comment:
Can we keep the same convention as above? i.e. `isTopBoxadApplicable`
Or change above method to `*Enabled`

Reply:
Fixed